tools/init_paths.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# exit immediately on failure
+set -e
+
+if [ -z "$TOOLS_DIR" ]; then
```

Comment:
It looks like callers are already effectively required to set `TOOLS_DIR` to something